core/src/main/scala/kafka/tools/ConsoleConsumer.scala

```diff
@@ -315,11 +321,21 @@ object ConsoleConsumer extends Logging {
 
     formatter.configure(formatterArgs.asScala.asJava)
 
-    val topicOrFilterOpt = List(topicOpt, whitelistOpt).filter(options.has)
-    if (topicOrFilterOpt.size != 1)
-      CommandLineUtils.printUsageAndDie(parser, "Exactly one of whitelist/topic is required.")
+    val filterOpts = List(includeOpt, whitelistOpt).filter(options.has)
+    if (filterOpts.size == 2) {
+      CommandLineUtils.warn("--whitelist is DEPRECATED and ignored if --include specified.")
+    }
+
     topicArg = options.valueOf(topicOpt)
-    whitelistArg = options.valueOf(whitelistOpt)
+    includedTopicsArg = if (options.has(includeOpt))
+      options.valueOf(includeOpt)
+    else
+      options.valueOf(whitelistOpt)
+
+    val topicOrFilterOptArgs = Seq(topicArg, includedTopicsArg).filterNot(value => value == null)
+
+    if (topicOrFilterOptArgs.size != 1)
+      CommandLineUtils.printUsageAndDie(parser, "Exactly one of include/topic is required. whitelist is DEPRECATED use --include instead.")
```

Comment:
maybe we can check `options.has(whitelistOpt)` here and only print the deprecation message if they provide the deprecated argument?